frontend/src/views/Market.vue

```diff
@@ -955,360 +957,360 @@ export default Vue.extend({
           id: 0,
           type: 'CharacterRenameTag',
           nftPrice: 0.1,
-          name: 'Rename Tag',
-          description: 'Renames one character.',
+          name: (this as any).$t('market.nftList.characterRenameTag'),
```

Comment:
Is it possible to use `i18n.t` (like in App.vue)  instead of `(this as any).$t`? Just to avoid using `any` where it's not necessary

Reply:
Done in 34a24aa and 91487d5